plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -483,15 +483,13 @@ func ClusterRoles() []rbac.ClusterRole {
 	}
 
 	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeScheduling) {
-		// Find the scheduler role
-		for i, role := range roles {
-			if role.Name == "system:kube-scheduler" {
-				pvRule := rbac.NewRule("update").Groups(legacyGroup).Resources("persistentvolumes").RuleOrDie()
-				scRule := rbac.NewRule(Read...).Groups(storageGroup).Resources("storageclasses").RuleOrDie()
-				roles[i].Rules = append(role.Rules, pvRule, scRule)
-				break
-			}
-		}
+		roles = append(roles, rbac.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{Name: "system:kube-volume-scheduler"},
+			Rules: []rbac.PolicyRule{
+				rbac.NewRule("update").Groups(legacyGroup).Resources("persistentvolumes").RuleOrDie(),
```

Comment:
include `get`,`list`,`watch`, `update`, `patch`, to make this role sufficient for all the read/update operations you'd need to do on a PV

Reply:
done